.gitignore

```diff
@@ -7,3 +7,4 @@ node_modules
 *.map
 *.log
 *.sublime*
+README.md
```

Comment:
I don't think this was an intended change as we actively want people updating the README list as they submit PRs.